Reflections/ReflectionWeek4.md

```diff
@@ -0,0 +1,17 @@
+1. For this week I didn't accomplish that much as I have been busy with midterms.
+- I was able to get a working screen color changing slider so the user can actually make the design of their app be whatever they want because I wanted the color wheel to represent the different colors the app can be.
+- I didn't get finished but I recently started using the free to use editing application because I didn't have the money for Adobe After Effects. I used this to begin creating the animation for the logo when the app starts up.

```

Comment:
I believe you do not have to buy Adobe After Effects, you can get it installed in your computer through Berea. If you are on campus this should be as easy as booking an appointment with IS&S here: https://bereacollegeiss.setmore.com/ and if you are off campus you can book a virtual appointment and have them do it remotely over zoom.